stubs/google/appengine/api/__init__.pyi

```diff
@@ -0,0 +1,3 @@
+from typing import Any
+
+datastore_types: Any = ...
```

Comment:
Can we get rid of this `Any`?

Reply:
Actually datastore_types is not needed. Removing. Thanks!

Reply:
Actually this is not needed. Deleted. Thanks!